ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3Subsystem50Parser.java

```diff
@@ -110,7 +110,7 @@ private void parseApplicationSecurityDomains(final XMLExtendedStreamReader reade
         }
     }
 
-    private void parseApplicationSecurityDomain(final XMLExtendedStreamReader reader, final List<ModelNode> operations) throws XMLStreamException {
+    protected void parseApplicationSecurityDomain(final XMLExtendedStreamReader reader, final List<ModelNode> operations) throws XMLStreamException {
```

Comment:
looks like only this method is overridden in subclass, so we probably don't need `protected` access modifier for other methods.

Reply:
Fixed